de/_posts/2019-12-07-rails-beginner-workshop.markdown

```diff
@@ -4,5 +4,15 @@ tags: beginner rails_beginner_workshop
 slug: Anfänger-Workshop
 i18n_key: rails_beginner_workshop_2019_12
 ---
+![](../../../../../../assets/images/Dec19-Banner.jpeg)
```

Comment:
`![](/assets...)` is all you need

Reply:
Yeap, I realized reading your last PR! Will change it :)